docs/Installation.md

```diff
@@ -1,58 +1,135 @@
+# Installation
+
 T4 has two different components:
-* A Python package, `t4`, distributed via `pip`
-* A web interface, the T4 Catalog, deployed using [AWS CloudFormation](https://aws.amazon.com/cloudformation/)
+* A Python package, `t4`
+* A web interface, the T4 Catalog
 
 If you have an already-provisioned catalog, only the Python package is needed.
 
+## T4 client (Python 3.6)
 
-## Installing the Python package
+```bash
+$ pip install t4
+```
 
-T4 is currently in pre-release alpha. To install the current T4 `master`, first ensure that you are in a Python 3.6 environment, then run:
+If you wish to use AWS resources, such as S3 buckets, you will need valid AWS credentials. If this is your first time using the AWS CLI, run the following:
 
 ```bash
-$ pip install git+https://github.com/quiltdata/t4.git#subdirectory=api/python
+$ pip install aws-cli
+$ aws configure
 ```
 
-In order to interface with a remote catalog you will also need valid AWS credentials. If this is your first time using AWS, ensure that you have the AWS CLI installed (`pip install aws-cli`), then run the following to store the IAM credentials you wish to use with T4:
+If you are already using the AWS CLI, you may use your existing profile, or [create a new profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html).
+
+### Developer
+
+Install the current T4 client from `master`:
 
 ```bash
-$ aws configure
+$ pip install git+https://github.com/quiltdata/t4.git#subdirectory=api/python
 ```
 
-If you are already using AWS IAM you may reuse your credentials for T4. Alternatively you can create a Quilt-specific [profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html).
+## Catalog
 
+The Javascript catalog is a single-page, static application.
```

Comment:
Is the word static meaningful here? I think just "single page application" is the keyword you want.

Reply:
Static is important because there is no webserver required